packages/metrics/package.json

```diff
@@ -88,7 +88,8 @@
     "url": "https://github.com/aws-powertools/powertools-lambda-typescript/issues"
   },
   "dependencies": {
-    "@aws-lambda-powertools/commons": "2.27.0"
+    "@aws-lambda-powertools/commons": "2.27.0",
+    "@aws/lambda-invoke-store": "0.1.0"
```

Comment:
Since we added this as dependency of the `commons` pkg, do we need to declare it here again?

Reply:
Oh that's a good point, we may not. Let me play around with it.